app/views/organizations/index.html.erb

```diff
@@ -1,41 +1,51 @@
-<div>
-  <div id="map" style='height: 280px;'></div>
-</div>
+<div class = "header-container" >
+  <div class = "video-container">
+    <video preload="true" autoplay loop muted>
+      <source src="<%= video_path('skyline_clip.mp4') %>" type="video/mp4">
+      </video>
+    </div>
+    <h2>  VANCOUVER TECH HUB </h2>
```

Comment:
Space after `<h2>` and before `VANCOUVER...` is unnecessary. Same for the same space that appears before `</h2>`